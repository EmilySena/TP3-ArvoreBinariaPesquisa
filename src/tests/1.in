i zhita
i ixthanawe
i hawzxaheas
i azitar
i azzahawshx
i alskehaskalee
i mianne
i shultzzza
i uet
i nhanhueaz
i zulhan
e 5 zhita zhita zulhan uet azitar
d 2 1 11
s zhita ulman
s ulman zhita
e 5 zhita zhita zulhan uet azitar
e 3 mianne uet azzahawshx
d 3 2 1 1
s mianne zhulz
e 3 zhulz zhita zulhan
