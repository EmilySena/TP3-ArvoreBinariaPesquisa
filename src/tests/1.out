1 1 11 10 4
zhita zulhan
11 11 10 1 4
7 1 6
ixthanawe uet uet
11 10 9
